code/modules/mob/living/carbon/carbon_movement.dm

```diff
@@ -56,8 +56,6 @@
 			nutrition -= HUNGER_FACTOR/10
 			if(m_intent == MOVE_INTENT_RUN)
 				nutrition -= HUNGER_FACTOR/10
-		if((disabilities & FAT) && m_intent == MOVE_INTENT_RUN && bodytemperature <= 360)
```

Comment:
accidental removal?

Reply:
No, completely intentional. It shortcircuits from the disabilities & FAT check but this proc gets called every time a carbon moves and fat people exist for maybe one minute, once in every 50 rounds since eating is completely optional. So why bother?

I can take it out if it's actually a feature anyone wants in the codebase.

< Naksu> korphaeron: is fat people getting hot when running a feature we really need in /mob/living/carbon/Move
< Naksu> especially given that a round very likely doesn't have fat people in it
<@korphaeron> probably not naksu